pkg/appconsts/testground/app_consts.go

```diff
@@ -0,0 +1,7 @@
+package v1
```

Comment:
[blocking] I find it confusing that this package statement uses `v1` when this file isn't located it in the `v1` directory. Did this mean to say:

```suggestion
package testground
```